fdbcli/SetClassCommand.actor.cpp

```diff
@@ -103,8 +116,8 @@ ACTOR Future<bool> setClassCommandActor(Reference<IDatabase> db, std::vector<Str
 	} else if (tokens.size() == 1) {
 		wait(printProcessClass(db));
 	} else {
-        bool successful = wait(setProcessClass(db, tokens[1], tokens[2]));
-        return successful;
+		bool successful = wait(setProcessClass(db, tokens[1], tokens[2]));
+		return successful;
```

Comment:
```suggestion
		return wait(setProcessClass(db, tokens[1], tokens[2]));
```

Reply:
Done. Thanks

Reply:
Ah yeah my bad, looks like this is not possible.